.storybook/preview.tsx

```diff
@@ -17,5 +16,10 @@ export const parameters = {
   },
 };
 
-// @storybook/addon-storyshots isn't successfully injecting the decorators from storybook-addon-react-router-v6 and storybook-addon-apollo-client so we need to manually wrap the stories.
-export const decorators = !isTest ?  [withRouter] : [(Story) => <MockedProvider><MemoryRouter>{<Story />}</MemoryRouter></MockedProvider>];
+export const decorators = [
```

Comment:
Adding the GlobalStyles decorator should fix the fonts not rendering. We do this in parsley.
https://github.com/evergreen-ci/parsley/blob/5d2269e8245499a286c84edc041a6383eaf94ec3/src/components/styles/GlobalStyles.tsx#L1

Reply:
See above, this fixes the issue locally but not remotely.